samples/nrf9160/nrf_cloud_mqtt_multi_service/src/application.c

```diff
@@ -23,42 +26,48 @@ LOG_MODULE_REGISTER(application, CONFIG_MQTT_MULTI_SERVICE_LOG_LEVEL);
 /* Timer used to time the sensor sampling rate. */
 static K_TIMER_DEFINE(sensor_sample_timer, NULL, NULL);
 
+/* Buffer to contain modem responses when performing AT command requests */
+static char at_req_resp_buf[CONFIG_AT_CMD_REQUEST_RESPONSE_BUFFER_LENGTH];
+BUILD_ASSERT(CONFIG_AT_CMD_REQUEST_RESPONSE_BUFFER_LENGTH >= 40,
```

Comment:
` "Error while processing AT command request: %d"` is larger than 40... #define the string and use sizeof here?

Reply:
Great catch. I'll do exactly as you suggest